lib/MusicBrainz/Server/Data/Role/IPI.pm

```diff
@@ -21,6 +21,31 @@ role
         value_type          => 'ipi',
     };
 
+    method find_reused_ipis => sub {
+        my ($self, @ipis) = @_;
+        my $query = "SELECT 'artist' AS entity_type, artist_ipi.ipi, COUNT(*) AS count
+                    FROM artist
+                        JOIN artist_ipi ON artist.id = artist_ipi.artist
+                    WHERE artist_ipi.ipi = any(?)
+                    GROUP BY artist_ipi.ipi
+                    UNION
```

Comment:
```suggestion
                    UNION ALL
```
UNION ALL should be used if uniqueness isn't an issue, 'cause UNION is slower (and there shouldn't be duplicates here).